internal/iam/repository_role_grant.go

```diff
@@ -610,3 +610,65 @@ func (r *Repository) grantsForUserGlobalResources(
 
 	return ret, nil
 }
+
+// grantsForUserOrgResources returns the grants for the user for resources that can
+// only be org scoped.
+func (r *Repository) grantsForUserOrgResources(
+	ctx context.Context,
+	userId string,
+	res resource.Type,
+	reqScope Scope,
+	opt ...Option,
+) (perms.GrantTuples, error) {
+	const op = "iam.(Repository).grantsForUserOrgResources"
+	if userId == "" {
+		return nil, errors.New(ctx, errors.InvalidParameter, op, "missing user id")
+	}
+	requestScope := reqScope.GetPublicId()
+	if requestScope == "" {
+		return nil, errors.New(ctx, errors.InvalidParameter, op, "missing request scope id")
+	}
```

Comment:
nitpick (sorry, missed this one in my last review)
```suggestion
	if  reqScope.GetPublicId() == "" {
		return nil, errors.New(ctx, errors.InvalidParameter, op, "missing request scope id")
	}
```

Reply:
I'm passing `requestScope` to `args` later on in this function -- I figured I'd just store the value of `reqScope.GetPublicId()` since I'm calling it more than once